### DISCUSSION TITLE
fail2ban配置的ssh登录策略无效

### DISCUSSION THREAD

**Discussion by sunhaoee** [1Panel-dev/1Panel#5597] [Q&A]
### 1Panel 版本

v1.10.10-lts

### 问题描述

系统是 Ubuntu 22.04.4 LTS
已经在fail2ban设置了ssh登录的策略，但是依旧有ip能多次尝试，并且fail2ban中没有ban的日志
![image](https://github.com/1Panel-dev/1Panel/assets/32056362/70afdc1b-2b36-4f82-b284-435eab7dfb7a)

![image](https://github.com/1Panel-dev/1Panel/assets/32056362/4adec11f-d606-46ac-a8f1-33c1a53d42b1)
![image](https://github.com/1Panel-dev/1Panel/assets/32056362/cd3cd3a2-072b-437b-8e84-b5d3059b26b6)

### 重现步骤

通问题描述

**Comment by ssongliu** [comment]
感谢反馈，检查一下 ufw 防火墙状态是否正常  systemctl status ufw

**Comment by sunhaoee** [comment]
> 感谢反馈，检查一下 ufw 防火墙状态是否正常 systemctl status ufw

ufw状态是正常的

![image](https://github.com/1Panel-dev/1Panel/assets/32056362/60aa9510-012a-4406-adbb-b19981aad39e)

**Comment by ssongliu** [comment]
> > 感谢反馈，检查一下 ufw 防火墙状态是否正常 systemctl status ufw
> 
> ufw状态是正常的
> 
> ![image](https://github.com/1Panel-dev/1Panel/assets/32056362/60aa9510-012a-4406-adbb-b19981aad39e)

ssh 端口是 22 吗

**Comment by sunhaoee** [comment]
> > > 感谢反馈，检查一下 ufw 防火墙状态是否正常 systemctl status ufw
> > 
> > 
> > ufw状态是正常的
> > ![image](https://github.com/1Panel-dev/1Panel/assets/32056362/60aa9510-012a-4406-adbb-b19981aad39e)
> 
> ssh 端口是 22 吗

是22，
auth.log的结构是这样的，不知道有没有关系
![image](https://github.com/1Panel-dev/1Panel/assets/32056362/f0a9300c-3519-4810-a336-556eca26cae8)

**Comment by ssongliu** [comment]
麻烦再提供一下  ufw status verbose 截图

**Comment by sunhaoee** [comment]
> 麻烦再提供一下 ufw status verbose 截图

![image](https://github.com/1Panel-dev/1Panel/assets/32056362/61d3864f-5db7-4bb3-b509-426a110a796d)

**Comment by ssongliu** [comment]
麻烦你加一下我吧，我远程看看
![image](https://github.com/1Panel-dev/1Panel/assets/73214554/bc52347c-148b-4152-804b-06308b0691dc)
